src/ome_zarr_models/v04/coordinate_transformations.py

```diff
@@ -26,7 +27,18 @@ class VectorScale(Base):
     """
 
     type: Literal["scale"]
-    scale: list[float] = Field(..., min_length=2)
+    scale: list[float]
+
+    @classmethod
+    def build(cls, data: Iterable[float]) -> Self:
+        """
+        Create a VectorScale from an iterable of floats.
+        """
+        return cls(type="scale", scale=tuple(data))
+
+    @property
+    def ndim(self) -> int:
```

Comment:
Nice, I like this! Where we think it's sensible sprinling useful properties like this will make the library way more useful.